docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,415 @@
+---
```

Comment:
General advice:

Tell the user what to do, remove "so" and other filler words.  For example,

```
So you need to foo the bar.
```

change to

```
Foo the bar.
```